### PR TITLE
S02 subscribe

### PR DESCRIPTION
**Pull request by SebSamPro**

### CODE REVIEW

File: frontend/src/assets/arrowBackDark.png
Comment (ItsMatDev) [inline frontend/src/assets/arrowBackDark.png:1]: remove png and keep only svg

**Review by ItsMatDev — APPROVED** [review]

File: frontend/src/components/BackButton/Redirection.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { useNavigate } from "react-router-dom";
+
+function Redirection() {
+  const navigate = useNavigate();
+
+  return (
+    <button type="button" onClick={() => navigate(-1)}>
```
Comment (Yoann-Fortin) [inline frontend/src/components/BackButton/Redirection.jsx:7]: Isn't a button but a link

File: frontend/src/pages/Subscribe/Subscribe.jsx
Comment (Yoann-Fortin) [inline frontend/src/pages/Subscribe/Subscribe.jsx:1]: Wrong semantic

File: frontend/src/pages/Subscribe/Subscribe.scss [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  bottom: 1rem;
+}
+
+.addPicture {
+  width: 6rem;
+  height: auto;
+}
+
+.inputPictureProfil {
+  display: none;
```
Comment (Yoann-Fortin) [inline frontend/src/pages/Subscribe/Subscribe.scss:120]: Why display none ?

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**